activesupport/test/core_ext/string_ext_test.rb

```diff
@@ -56,39 +56,37 @@ def test_strip_heredoc_on_a_regular_indented_heredoc_with_blank_lines
     EOS
   end
 
-  def test_pluralize
-    SingularToPlural.each do |singular, plural|
-      assert_equal(plural, singular.pluralize)
-    end
+  test "pluralize", each: SingularToPlural do |singular, plural|
+    assert_equal plural, singular.pluralize
```

Comment:
I do think that retaining spacing and parenthesis usage will help to shrink this diff to its most important parts, and will facilitate review from Core or a Committer:

```suggestion
    assert_equal(plural, singular.pluralize)
```

Reply:
Done